File: back-end/src/main/java/com/csulb/cookie/domain/Customer.java
package com.csulb.cookie.domain;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import java.io.Serializable;
import lombok.Data;

@TableName(value ="customer")
@Data
public class Customer implements Serializable {

    @TableId(type = IdType.AUTO)
    private Integer id;
    private String username;
    private String nickName;
    private String password;
    private String email;
    private String phone;

}

File: front-end/lib/domain/customer.dart
class Customer {
  int? id;
  String username;
  String nickName;
  String password;
  String email;
  String phone;

  Customer(this.id, this.username, this.nickName,this.password, this.email, this.phone);

  @override
  String toString() {
    return 'Customer{username: $username, nickName: $nickName, password: $password, email: $email, phone: $phone}';
  }
}